### DISCUSSION TITLE
Which Asian fruit - nicknamed the "King of Fruits" - is known for its smell?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#456] [Galaxy Trivia] [answered]
**Bonus:** And which one is nicknamed the "Queen of Fruits"?

**Comment by ParanoidUser** [comment]
@skzidan, @newzpanuwat, @muzammilali28, @trevorwang, @mayankjaiswal1709 any clue? 🥭🍈

**Reply by 0x4248** [reply]
The Asian fruit that is nicknamed the "King of Fruits" and is known for its distinctive smell is durian. Durian is a large, spiky fruit that is native to Southeast Asia and is particularly popular in countries like Thailand, Malaysia, and Indonesia. While some people find the aroma of durian to be unpleasant or even offensive, others consider it to be sweet and alluring. The flesh of the fruit is yellow and custard-like in texture, and it is often used in a variety of sweet and savory dishes.

**Reply by ParanoidUser** [reply]
Well done! Unfortunately, there is no way to mark the reply "as answer". Next time, please post it as a new suggestion in a thread.

**Reply by 0x4248** [reply]
oh ok

**Comment by thuongtruong1009** [comment, marked as answer]
The Asian fruit that is nicknamed the "King of Fruits" and known for its distinctive smell is the durian. Durians are native to Southeast Asia and are known for their spiky exterior, creamy flesh, and pungent odor, which some people describe as being similar to rotting onions or sweaty socks. Despite its strong smell, the durian is considered a delicacy in many parts of Asia and is used in a variety of sweet and savory dishes. However, the fruit is banned in some public places and hotels due to its strong odor.
